int64[] snack_ids
---